scripts/pugbomb.py

```diff
@@ -2,19 +2,29 @@
 # -*- coding: utf-8 -*-
 #Created by Harsha Goli
 import praw
+import re
 
 #pug bombs the chat and destroys this poor bot's soul
 def pugbomb(response, args):
     #gets the number
     num = int(args[0])
     if num > 10:
         num = 10
+
     reddit = praw.Reddit(client_id='aGpQJujCarDHWA',
                      	client_secret='fkA9lp0NDx23B_qdFezTeGyGKu8',
                      	user_agent='my user agent',
                         password='PHGSU2017',
                         username='Panther_Bot')
+    
     pug_urls=[]
     for submission in reddit.subreddit('pugs').hot(limit=num):
+        if 'imgur' in submission.url and not '.jpg' in submission.url and not '.png' in submission.url:
```

Comment:
Cool, one last question, before we were also handling jpeg with the regex, do we not want to do that?